x-pack/plugins/security_solution/public/timelines/components/open_timeline/helpers.ts

```diff
@@ -257,6 +257,7 @@ export const defaultTimelineToTimelineModel = (
   const timelineEntries = {
     ...timeline,
     columns: timeline.columns != null ? timeline.columns.map(setTimelineColumn) : defaultHeaders,
+    defaultColumns: defaultHeaders,
```

Comment:
This was needed in order to reset the timeline page columns property, if not defined the reset action updates to empty columns.

Reply:
This also resolves @andrew-goldstein comment https://github.com/elastic/kibana/pull/105207#pullrequestreview-708791344